extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -539,6 +545,12 @@ class QueueProcessor {
                             error: err.description });
                 return done();
             }
+            if (err.ObjNotFound) {
+                log.info('replication skipped: ' +
```

Comment:
It's actually not an error, just a singular case of not replicating anything. (the user can create and remove an object version on the source before the queue populator has a chance to look at it, it's a normal situation).